src/makei/ibmi_json.py

```diff
@@ -49,7 +49,7 @@ def from_file(cls, file_path: Path, parent_ibm_i_json: "IBMiJson") -> "IBMiJson"
     def __dict__(self):
         build = {}
 
-        if self.build["tgt_ccsid"] != DEFAULT_TGT_CCSID and self.build["tgt_ccsid"] != "":
+        if self.build["tgt_ccsid"] != DEFAULT_TGT_CCSID and self.build["tgt_ccsid"] != None:
```

Comment:
probably should guard against either, if it is possible to store "" as a value in Json then we should not consider the value specified